Src/CSharpier.Tests/TestFiles/FieldDeclaration/FieldDeclarations.cst

```diff
@@ -17,8 +17,7 @@ public class ClassName
     private SomeObject SomeLongerName = new SomeObject(
         "lkjasdflkjasdfkljaskldjf",
         "klasldkfaksdfasdfkjasdklf"
-    )
-    {
```

Comment:
I kind of hate how C# creators picked the stupid break-before-curly pattern with no thought for readability just to be different from Java. It's forcing us to be so inconsistent with when we use space brace. :pouting_cat:

Reply:
Yeah I kind of want csharpier to not break-before-curly, but am worried that would turn off a lot of people.
I think things are starting to get more consistent, but there are definitely still some edge cases that need to get handled.